### DISCUSSION TITLE
wiki.js : two factor authentication recovery

### DISCUSSION THREAD

**Discussion by bertiebw** [requarks/wiki#6630] [Help / Questions] [answered]
Good afternoon,
My question is about two factor authentication in wiki.js.
I read the documentation https://docs.requarks.io/auth before posting the question.
My intention is to start wiki.js with one user only and two factor authentication.
If I enable two factor authentication and I am the only user, how can I recover my authentication settings if I lose my mobile for example?
What recovery method is in place please?
Thanks.
Bertie

**Comment by NGPixel** [comment, marked as answer]
As long as you have DB access, you can disable 2FA on your user and set it up again.
